src/rnp/rnpcfg.c

```diff
@@ -306,17 +306,12 @@ bool
 rnp_cfg_getbool(rnp_cfg_t *cfg, const char *key)
 {
     const char *val = rnp_cfg_get(cfg, key);
-
-    if (val) {
-        if ((strcmp(val, "true") == 0) || (strcmp(val, "True") == 0)) {
-            return true;
-        } else if (atoi(val) > 0) {
-            return true;
-        } else
-            return false;
-    } else {
-        return false;
+    
+    if (val && ((strcmp(val, "true") == 0) || (strcmp(val, "True") == 0) || (atoi(val) > 0))) {
```

Comment:
better to use ``tolower()`` and then compare to "true"

Reply:
Agree.

Reply:
Used strcasecmp - otherwise would need to check val, allocate buffer for lowercase copy, deallocate, etc, etc.